packages/react-components/react-persona/src/stories/Persona/PersonaDefault.stories.tsx

```diff
@@ -0,0 +1,4 @@
+import * as React from 'react';
+import { Persona, PersonaProps } from '@fluentui/react-persona';
```

Comment:
```suggestion
import { Persona } from '@fluentui/react-persona';
import type { PersonaProps } from '@fluentui/react-persona';
```

Reply:
Sorry I just saw the comment, I'll update it in a follow-up PR